tests/data/EventFactory.ts

```diff
@@ -24,21 +24,45 @@ export class EventFactory {
 
   public static fake(): EventModel {
     const [start, end] = EventFactory.randomTime();
+    return EventFactory.fakeWithTime(start, end);
+  }
+
+  private static fakeWithTime(start: Date, end: Date): EventModel {
```

Comment:
You can use the `with` method instead of `fakeWithTime`, e.g.
```typescript
private static fakeWithTime(start: Date, end: Date): EventModel {
  return with({ start, end });
}
```

Reply:
Totally forgot about `with()` - we actually don't even need `fakeWithTime` or `fakeWithoutTime` anymore if `with()` is used for `fakePastEvent(), fakeOngoingEvent(),` and `fakeFutureEvent()`.